### ISSUE TITLE
Different behaviour from ResponsiveContainer

### ISSUE THREAD

**Issue by vkallore** [recharts/recharts#1070]
## Do you want to request a feature or report a bug?
Bug
## What is the current behavior?
If included ResponsiveContainer, it expects it's parent container to have a fixed height (directly under _container_ div).
http://jsfiddle.net/v8gxrbyu/236/
Non working example, parent container of ResponsiveContainer do not have it's height defined.
http://jsfiddle.net/v8gxrbyu/237/
To make the above non working to work, I have to give fixed height and width to its container, and now its working.
http://jsfiddle.net/v8gxrbyu/239/
## What is the expected behavior?
I believe, we do not have to explicitly mention parent container's height. Or is it expecting due to _absolute_ positioning?
## Which versions of Recharts, and which browser / OS are affected by this issue? Did this work in previous versions of Recharts?
Used latest version 1.0.0-beta.6. Tried it in Ubuntu 14.04, Chromium 58.0.3029.110

**Comment by zlanich** [comment]
+1 for me. I'm not sure how to use the ResponsiveContainer to my advantage with its current implementation.

**Comment by xile611** [comment]
Sorry for not handling your issue in time, please try the latest version. If the problem persists, please open a new issue according to the issue guide.

**Comment by kvangrae** [comment]
I've encountered this issue and tested it against 1.8.5. I reproduced it in which my parent container had a percentage for the height. However, in some instances it would display just fine, and in others it wouldn't display. But, in the instances where it wouldn't display, it consistently failed and vice versa. Also, this issue only appeared in Firefox. I was using Firefox 68.10.0esr. Chrome did not show the issue.

I'm going to work around it by setting a fixed height for now.

**Comment by kvangrae** [comment]
To elaborate a bit further, I found that having a *min-height* in my parent container caused my issue:
```css
.parent {
  height: 22%;
  min-height: 180px;
}
```
If I removed *min-height* from the parent component, the chart rendered as expected.

To summarize, I have found two workarounds:

1. For the parent component, use fixed *height* instead of percentages.
2. For the parent component, if using percentage *height*, do not use *min-height*.
